samples/openthread/cli/README.rst

```diff
@@ -121,6 +121,9 @@ To activate the optional extensions supported by this sample, modify :makevar:`O
 * For the minimal single protocol variant, set :file:`overlay-minimal_singleprotocol.conf`.
 * For the minimal multiprotocol variant, set :file:`overlay-minimal_multiprotocol.conf`.
 * For USB transport support, set :file:`overlay-usb.conf`. Additionally, you need to set :makevar:`DTC_OVERLAY_FILE` to :file:`usb.overlay`.
+* For turning on logging set :file:`overlay-logging.conf`.
+* For redirecting logs to RTT set :file:`overlay-rtt.conf`.
```

Comment:
Could you additionally link RTT to its documentation: https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/ug_logging.html?highlight=rtt%20logging#rtt ?

Reply:
fixed.